### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Goruchie**
## **REQUIREMENTS**

- **Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder. The index.html file must be set as a template using the HTML Webpack Plugin.**
- **Create an index.js file and set an array of some simple to do tasks (array of objects).**
-  **Write a function to iterate over the tasks array and populate an HTML list item element for each task.**
-  **On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.**
-  **Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.**
-  **All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
-# To-Do-list-list-structure
\ No newline at end of file
+# To Do List
+
+In this project, I built a simple HTML list of To Do tasks. This simple web page will be built using webpack and served by a webpack dev server.
+
+
+## Built With
+
```
Comment (omar-muhamad) [inline README.md:7]: - [ ] Please add the `Getting Started` section with the instructions on how to clone your repo, install dependencies libraries `npm install`, run your build `npm run build`, and run your dev server` npm run start`. To make your readme file professional and user-friendly.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Goruchie 

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -1 +1,72 @@
...
+### Setup
+[The setup of webpack](https://webpack.js.org/guides/getting-started/)
+
+### Install
+
+- A code editor (e.g. VSC) /text editor (e.g. notepad++)
+
+- Node.js
+
+## Authors
```
Comment (rloterh) [inline README.md:46]: - [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

File: README.md [hunk 3]
```diff
@@ -1 +1,72 @@
...
+
+- HTMl
+- Webpack
+- CSS
+- Javascript
+
+## Live Demo (if available)
+
+## Getting Started
+
```
Comment (rloterh) [inline README.md:16]: - [ ] OPTIONAL
Also kindly include in your ReadMe the _Getting Started_ section,  instructions on how to **run** your application.

File: README.md [hunk 4]
```diff
@@ -1 +1,72 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rloterh) [inline README.md:72]: - [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Marcos,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- Project is set up correctly
- Webpack is correctly configured


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_

keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**